helm-charts/konk-service/templates/kubeconfig-deployment.yaml

```diff
@@ -47,19 +47,28 @@ spec:
               kubectl config set-context {{ .Values.konk.name }} --cluster {{ .Values.konk.name }} --user kubernetes-admin
               kubectl config use-context {{ .Values.konk.name }}
 
-              # these commands run in infra cluster
+              # the following commands run in infra cluster
               unset KUBECONFIG
+
+              # deletes existing kubeconfig after a cert change
+              if ! md5sum --status --check /tmp/certsum
+              then
+                kubectl -n $NAMESPACE delete secret {{ include "konk-service.fullname" . }}-kubeconfig
```

Comment:
Will replacing the secret with a different one with the same name work for pods that already have it mounted? Or should I patch the existing one instead?

Reply:
Fixed.